scss/civicrm/contact/pages/_contributions.scss

```diff
@@ -1013,9 +1013,9 @@ form.CRM_Contribute_Form_ContributionCharts {
   .form-layout-compressed {
     margin: 20px 0 0;
 
-    input.crm-form-submit {
+    button.crm-form-submit {
       margin-left: 5px;
-      padding: 4px 10px;
+      padding: 8px 11px;
```

Comment:
Just a question here, not an action item, but why do we need these very specific padding values? Why not remove them and let the `.crm-form-submit` class from `_button.scss` do it's job? And why such specific values?

Reply:
Unfortunately paddings are different in different buttons, and right now there is no standard way to do this just by using classes
![2020-10-20 at 1 00 PM](https://user-images.githubusercontent.com/5058867/96554585-57c7a300-12d4-11eb-94e9-ae8c5ba68d53.png)

Reply:
Thanks @deb1990 I think this is something we might need to look into the future and find out why they behave this way. A PR to CiviCRM-core might be needed to keep them standard.